ethstore/src/account/kdf.rs

```diff
@@ -26,7 +26,7 @@ pub struct Pbkdf2 {
 	pub c: u32,
 	pub dklen: u32,
 	pub prf: Prf,
-	pub salt: [u8; 32],
+	pub salt: Vec<u8>,
```

Comment:
why did you change it to a `Vec<u8>`? I see no reason to do that

Reply:
@tworec just revert the changes

Reply:
but we do not the salt length, it can exceed 32 bytes. What then?

Reply:
All right, sorry did not see your comment there then it makes sense to use a vector instead of an array.

Reply:
👍